scripts/telegram_vc_adapter.py

```
@@ -51,38 +65,106 @@ def status(self) -> dict[str, Any]:
+    def pause(self) -> dict[str, Any]:
+        if self.chat_id is None:
+            raise AdapterError("Join a Telegram group call first")
+        if self.calls:
+            for method_name in ("pause_stream", "pause"):
+                fn = getattr(self.calls, method_name, None)
+                if callable(fn):
+                    try:
+                        fn(self.chat_id)
+                        break
+                    except Exception:
+                        pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)